### ISSUE TITLE
Add supporting for the screen of high dpi

### ISSUE THREAD

**Issue by chenyulue** [alejandroautalan/pygubu-designer#242]
Firstly, thanks for the awesome designer!

I use Win11, and the screen is of high dpi (2560 x 1600 pixels). I have set the screen scale to 150%. However, pygubu-designer seems not to support the screen of high dpi. 

The size of labels in the UI are **too small**, and the menus **overlap**, as the following shows.

 Are there any plans to add supporting for the screen of high dpi?

![2024-03-14_203554](https://github.com/alejandroautalan/pygubu-designer/assets/44137770/41a2cf47-2cf6-49ef-a9c8-9195179116cd)

**Comment by alejandroautalan** [comment]
Hello @chenyulue, thanks for trying pygubu.

I managed to recreate the problem. I will try to find a solution as soon as possible.

Regards
Alejandro A.

**Comment by MozisII** [comment]
Hi,
I resolved this issue by patching start of `main.py` starting line 56 (just after import section):
```
_# Initialize logger
logger = logging.getLogger(__name__)


# translator function
_ = translator

import ctypes
try: # >= win 8.1
    ctypes.windll.shcore.SetProcessDpiAwareness(2)
except: # win 8.0 or less
    _ctypes.windll.user32.SetProcessDPIAware()_

def init_pygubu_widgets():

    # Initialize all builders from plugins
    all_modules = []_
```

Regards
Sylvain

**Comment by chenyulue** [comment]
It solves the problem. Thanks! @MozisII

**Comment by alejandroautalan** [comment]
High dpi support implemented for Windows OS. Fixed in version 0.39
